server/server.js

```diff
@@ -6,28 +6,31 @@ const app = express();
 const chalk = require('chalk');
 const cookie = require('cookie-session');
 const GithubStrategy = require('./config/github')
-const {  db, VAPID_KEYS } = require('./config/credential.js');
+const { db } = require('./config/credential.js');
 const indexRoutes = require('./routes/index.routes');
 const expressValidator = require('express-validator');
 const methodOverride = require('method-override');
 const bodyParser = require('body-parser');
 const webPush = require('web-push');
-mongoose.connect(db, { useNewUrlParser: true })
-.then(()=> {
-  console.log(chalk.blue.bold("Database is connected"))
-})
-.catch((err)=> {
-  console.log(err)
-  console.log("Looks like MongoDB isn't installed yet!")
-  console.log(chalk.red.bold("Check it out, https://www.mongodb.com"))
-})
+mongoose.connect(db, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => {
+    console.log(chalk.blue.bold("Database is connected"))
+  })
+  .catch((err) => {
+    console.log(err)
+    console.log("Looks like MongoDB isn't installed yet!")
+    console.log(chalk.red.bold("Check it out, https://www.mongodb.com"))
+  })
 app.use(cors());
+
+const VAPID_KEYS = webPush.generateVAPIDKeys();

```

Comment:
@sastava007 Will, there be an issue since I think this will generate different vapid keys every time but VAPID keys should only be generated only once?

Reply:
Yess this will generate new VAPID_KEYS every time server is started. But this is the most common issue which every newbie is facing while setting up the project on their machine, so we can include this for now and will later remove it while merging with master branch.

Reply:
@sastava007 I don't think this is the correct way to solve this issue.